engine/chunk/src/main/java/io/deephaven/chunk/WritableChunkUtils.java

```diff
@@ -0,0 +1,275 @@
+//
+// Copyright (c) 2016-2025 Deephaven Data Labs and Patent Pending
+//
+package io.deephaven.chunk;
+
+import io.deephaven.util.QueryConstants;
+
+import java.util.Arrays;
+import java.util.Comparator;
+
+import static io.deephaven.util.QueryConstants.MIN_FINITE_DOUBLE;
+import static io.deephaven.util.QueryConstants.MIN_FINITE_FLOAT;
+import static io.deephaven.util.QueryConstants.NULL_CHAR;
+import static io.deephaven.util.QueryConstants.NULL_DOUBLE;
+import static io.deephaven.util.QueryConstants.NULL_FLOAT;
+
+final class WritableChunkUtils {
+
+    private static final int NULL_FLOAT_BITS = Float.floatToIntBits(NULL_FLOAT);
+    private static final int MIN_FINITE_FLOAT_BITS = Float.floatToIntBits(QueryConstants.MIN_FINITE_FLOAT);
+
+    private static final long NULL_DOUBLE_BITS = Double.doubleToLongBits(NULL_DOUBLE);
+    private static final long MIN_FINITE_DOUBLE_BITS = Double.doubleToLongBits(QueryConstants.MIN_FINITE_DOUBLE);
+
+    @SuppressWarnings("unused")
```

Comment:
Here and below. I'd rather not have an always-throws method in the codebase. I'm not sure why it needs to be here, but I'd like to enumerate my feelings.

1. If it doesn't need to be here, but you want to signal to other developers that you didn't forget the "boolean" case, then I think a comment might be more appropriate.
2. If it needs to be there to satisfy some codegen or dynamic linking case, can you please add a comment here and below that explains why these are marked unused and always throws?